lib/src/lib/components/form-fields/select-field/select-field.component.ts

```diff
@@ -179,7 +179,10 @@ export class SelectFieldComponent<T>
 
   public ngOnInit(): void {
     // load all options from the start
-    if (!this.options?.fetchOptionsOnFocus) {
+    if (
+      typeof this.options?.selectOptions !== 'function' ||
```

Comment:
Ignore als het niet relevant is maar selectOptions kan ook een observable zijn, mag die ook passeren?

Reply:
nvm dat is een object dan